Tests/Integration/DSCResources/McAfee.config.ps1

```diff
@@ -0,0 +1,79 @@
+configuration McAfee_config
+{
+    param
+    (
+
+        [Parameter()]
+        [AllowNull()]
+        [string]
+        $TechnologyRole,
+        
+        [Parameter()]
+        [AllowNull()]
+        [string]
+        $TechnologyVersion,
+
+        [Parameter(Mandatory = $true)]
+        [string]
+        $StigVersion,
+
+        [Parameter()]
+        [string[]]
+        $SkipRule,
+
+        [Parameter()]
+        [string[]]
+        $SkipRuleType,
+
+        [Parameter()]
+        [hashtable]
+        $Exception,
+
+        [Parameter()]
+        [hashtable]
+        $BackwardCompatibilityException,
+
+        [Parameter()]
+        [object]
+        $OrgSettings
+    )
+
+    Import-DscResource -ModuleName PowerStig
+
+    Node localhost
+    {
+        & ([scriptblock]::Create("
+        McAfee McAfeeConfiguration
+        {
+            Version = '$TechnologyVersion'
+            TechnologyRole = '$TechnologyRole'
+            Stigversion = '$StigVersion'
+            $(if ($OrgSettings -is [hashtable])
+            {
+                "Orgsettings = @{`n$($OrgSettings.Keys |
+                    ForEach-Object {"'{0}' = {1}{2} = '{3}'{4}`n" -f
```

Comment:
I know you copy/paste this as a template but style guidelines say to use named parameters so Foreach-Object is missing -Process